### PR TITLE
Renamed tests for cli/test_repository.py.

### PR DESCRIPTION
**Pull request by omaciel**
Test case naming follows #3038 conventions.

### CODE REVIEW

**Comment by omaciel** [comment]
```shell
(robottelo)┌[omaciel☮Ogs-MacBook-Pro.local]-(~/hacking/robottelo)-[git://rename-tests-cli-repository ✗]-
└> py.test -v tests/foreman/cli/test_repository.py --collect-only
================================================================= test session starts ==================================================================
platform darwin -- Python 2.7.10, pytest-2.8.3, py-1.4.31, pluggy-0.3.1 -- /Users/omaciel/.virtualenvs/robottelo/bin/python
cachedir: .cache
rootdir: /Users/omaciel/hacking/robottelo, inifile:
collected 21 items
Renamed tests for cli/test_repository.py.¬
<Module 'tests/foreman/cli/test_repository.py'>
  <UnitTestCase 'RepositoryTestCase'>
    <TestCaseFunction 'test_negative_create_with_name'>
    <TestCaseFunction 'test_positive_create_docker_repo_with_name'>
    <TestCaseFunction 'test_positive_create_docker_repo_with_upstream_name'>
    <TestCaseFunction 'test_positive_create_name'>
    <TestCaseFunction 'test_positive_create_name_label'>
    <TestCaseFunction 'test_positive_create_publish_via_http'>
    <TestCaseFunction 'test_positive_create_publish_via_https'>
    <TestCaseFunction 'test_positive_create_puppet_repo'>
    <TestCaseFunction 'test_positive_create_with_gpg_key_by_id'>
    <TestCaseFunction 'test_positive_create_with_gpg_key_by_name'>
    <TestCaseFunction 'test_positive_create_yum_repo'>
    <TestCaseFunction 'test_positive_create_yum_repo_with_checksum_type'>
    <TestCaseFunction 'test_positive_delete_by_id'>
    <TestCaseFunction 'test_positive_synchronize_docker_repo'>
    <TestCaseFunction 'test_positive_synchronize_yum_repo'>
    <TestCaseFunction 'test_positive_update_checksum_type'>
    <TestCaseFunction 'test_positive_update_gpg_key'>
    <TestCaseFunction 'test_positive_update_published_method'>
    <TestCaseFunction 'test_positive_update_url'>
    <TestCaseFunction 'test_positive_upload_content'>
    <TestCaseFunction 'test_verify_bugzilla_1189289'>

===================================================================  in 0.39 seconds ===================================================================
```

**Comment by sghai** [comment]
ACK.. Looks good.

File: tests/foreman/cli/test_repository.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def test_bugzilla_1189289(self):
 
     @run_only_on('sat')
     @tier1
-    def test_positive_create_1(self):
+    def test_positive_create_name(self):
```
Comment (oshtaier) [inline tests/foreman/cli/test_repository.py]: `test_positive_create_with_name`

File: tests/foreman/cli/test_repository.py [hunk 2]
```diff
@@ -101,7 +101,7 @@ def test_positive_create_1(self):
                 self.assertEqual(new_repo['name'], name)
 
     @tier1
-    def test_positive_create_2(self):
+    def test_positive_create_name_label(self):
```
Comment (oshtaier) [inline tests/foreman/cli/test_repository.py]: `test_positive_create_with_name_label` or `test_positive_create_with_name_and_label`(I prefer more second option, however first is more correct in terms of conventions)

File: tests/foreman/cli/test_repository.py [hunk 3]
```diff
@@ -122,7 +122,7 @@ def test_positive_create_2(self):
 
     @run_only_on('sat')
     @tier1
-    def test_positive_create_3(self):
+    def test_positive_create_yum_repo(self):
```
Comment (oshtaier) [inline tests/foreman/cli/test_repository.py]: `test_positive_create_with_yum_repo`

File: tests/foreman/cli/test_repository.py [hunk 4]
```diff
@@ -140,7 +140,7 @@ def test_positive_create_3(self):
                 self.assertEqual(new_repo['content-type'], u'yum')
 
     @tier1
-    def test_positive_create_4(self):
+    def test_positive_create_puppet_repo(self):
```
Comment (oshtaier) [inline tests/foreman/cli/test_repository.py]: `test_positive_create_with_puppet_repo`

**Comment by oshtaier** [comment]
ACK pending comments

**Comment by omaciel** [comment]
@oshtaier all issues addressed. Sorry about that, it was late in the evening and I let those slip :/

**Comment by oshtaier** [comment]
No need to say sorry and thanks for updates

**Comment by oshtaier** [comment]
APT

**Pull request merged**